examples/multi_gpu/distributed_sampling_xpu.py

```diff
@@ -103,6 +104,17 @@ def inference(
         return x_all
 
 
+@profileit('xpu')
```

Comment:
Is this desired? I would assume that profiling comes with an overhead.

Reply:
Yes, XPU profiling is lightweight, it brings `< 2%` overhead.